ext/hls/gsthlsdemux.c

```diff
@@ -115,9 +123,13 @@ static gboolean gst_hls_demux_get_live_seek_range (GstAdaptiveDemux * demux,
 static GstM3U8 *gst_hls_demux_stream_get_m3u8 (GstHLSDemuxStream * hls_stream);
 static void gst_hls_demux_set_current_variant (GstHLSDemux * hlsdemux,
     GstHLSVariantStream * variant);
+static void gst_hls_demux_get_property(GObject * object, guint prop_id,
+    GValue * value, GParamSpec * spec);
 
 #define gst_hls_demux_parent_class parent_class
-G_DEFINE_TYPE (GstHLSDemux, gst_hls_demux, GST_TYPE_ADAPTIVE_DEMUX);
+    G_DEFINE_TYPE(GstHLSDemux, gst_hls_demux, GST_TYPE_ADAPTIVE_DEMUX);
+
+const int RIXJOB_GSTHLSDEMUX_C_PATCH_VERSION = 1;
```

Comment:
By the way, if we control whole repository I think we no longer need these versions of each file. Maybe we can check commit or use only one variable per repo.